hyp3_autorift/vend/testautoRIFT.py

```diff
@@ -985,7 +986,7 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
                         parameter_file=kwargs['parameter_file'],
                     )
 
-                elif nc_sensor in ["L7", "L5", "L4"]:
+                elif nc_sensor in ("L7", "L5", "L4"):
```

Comment:
sicne this list won't change, tuple is better and more performant

Reply:
sounds good, thanks